python/lang/correctness/return-in-init.yaml

```diff
@@ -2,11 +2,13 @@ rules:
 - id: return-in-init
   patterns:
   - pattern-inside: |
-      class A(...):
+      class $A(...):
```

Comment:
We were only catching classes named `A` instead of using a metavariable :O

Reply:
Nice catch!